examples/custom-middleware/reject-machine-to-machine-traffic-using-custom-metadata-ACL-rules.md

```diff
@@ -18,7 +18,7 @@ We can also reject the specific request for example based on the custom metadata
           rejectBasicAuth: true
 ```
 
-2. Declare custom kibana js file `readonlyrest_kbn.kibana_custom_js_inject_file: '/path/to/custom_kibana.js'`
+2. Declare custom Kibana JS file `readonlyrest_kbn.kibana_custom_js_inject_file: '/path/to/custom_kibana.js'`
```

Comment:
specify it's injected in the head of the Kibana UI frontend code.

Nowadays we need to specify, because we support adding server side snippets (custom middleware) as well!

Reply:
@Dzuming maybe you will add it, because I don't feel it :)

Reply:
Added the info ✔️ 

One note, it's injected at the end of the HTML body tag of the Kibana UI to be precise :)